### PR TITLE
Adding healthscope to addon-local.

### PR DESCRIPTION
**Pull request by artursouza**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,15 +1,15 @@
-module github.com/crossplane/oam-controllers
+module github.com/crossplane/addon-oam-kubernetes-local
```
Comment (wonderflow) [inline go.mod]: Changing this means we must change repos which rely on this module. We'd better consider using a better name if we want to do this. What about `oam-core` ？

**Review by wonderflow — COMMENTED** [review]
Generally LGTM, please fix IC

File: go.mod (hunk 1 above)
Comment (ryanzhang-oss) [reply go.mod]: yeah, better to keep this name

**Review by ryanzhang-oss — CHANGES_REQUESTED** [review]
Change the module name will break any projects that rely on this

**Comment by artursouza** [comment]
Fixed the e2e tests by adding extra steps to install the latest CRDs from oam-kubernetes-runtime.

File: go.mod (hunk 1 above)
Comment (artursouza) [reply go.mod]: Done. Kept the same name.

**Review by wonderflow — APPROVED** [review]
LGTM, Good job!

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -52,10 +52,29 @@ jobs:
...
         kind get kubeconfig > kubeconfig
         export KUBECONFIG="$PWD/kubeconfig"
         kubectl wait -n kube-system --for=condition=Ready -l k8s-app=kube-dns pods --timeout=300s
         make e2e-setup
+
+    - name: Checkout oam-kubernetes-runtime.
+      uses: actions/checkout@v2
+      with:
+        repository: crossplane/oam-kubernetes-runtime
+        ref: 9d3a09f9c907e147393693fef561c6f389a025f2
```
Comment (ryanzhang-oss) [inline .github/workflows/go.yml:66]: are we checking out a specific version of the  oam-runtime?

File: Makefile [hunk 3]
```diff
@@ -58,12 +58,20 @@ docker-push:
 kind-load:
 	kind load docker-image $(IMG) || { echo >&2 "kind not installed or error loading image: $(IMG)"; exit 1; }
 
-e2e-setup:
+e2e-setup-master:
```
Comment (ryanzhang-oss) [inline Makefile]: where is this used?

**Review by ryanzhang-oss — CHANGES_REQUESTED** [review]
it seems that we need a way to solve the dependency between the addon and oam-runtime

File: Makefile (hunk 3 above)
Comment (artursouza) [reply Makefile]: It can be used later on. It is added on purpose.

File: .github/workflows/go.yml (hunk 2 above)
Comment (artursouza) [reply .github/workflows/go.yml:66]: Yes, to keep the build predictable.

**Comment by artursouza** [comment]
Added E2E tests for health scope. It needs the new appconfig controller in crossplane for one of the checks, so I will add that check once crossplane is updated.

**Review by ryanzhang-oss — APPROVED** [review]

**Pull request merged**